gradle.properties

```diff
@@ -7,4 +7,6 @@ signing.secretKeyRingFile=PathToYourKeyRingFile
 ossrhUsername=your-jira-id
 ossrhPassword=your-jira-password
 
-selenium.version=4.4.0
+selenium.version=4.5.0
+# Please increment the value in a release
+appiumClient.version=8.2.0
```

Comment:
cc @SrinivasanTarget 
I've moved this appiumClient version definition here to read the value for User Agent (in src/main/java/io/appium/java_client/AddAppiumUserAgent.java). You will need to increment here in a release